### PR TITLE
Specify a name for the logger created in main.

### PR DESCRIPTION
**Pull request by alikins**

### CODE REVIEW

File: galaxy_importer/main.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def main(args=None):
 
 def setup_logger(cfg):
     """Sets up logger with custom formatter."""
-    logger = logging.getLogger()
+    logger = logging.getLogger('galaxy_importer')
```
Comment (cutwater) [inline galaxy_importer/main.py:53]: Can you please give a rationale explaining why this change was made?

Comment (alikins) [reply galaxy_importer/main.py]: getLogger() in main.py will get the root logger. ie logger name=''.  That's not a useful name. It will also set the default/root logger if imported and called as a library potentially overriding logging config setup by the application. 

logging.getLogger('galaxy_importer') creates the main app logger and any other logger created with the normal 'logging.getLogger(__name__)' will live under the 'galaxy_importer' tree and by default use the level, filters, and handlers setup on the 'galaxy_importer' logger.

For logging from 'main.py', there is a 'galaxy_importer.main' setup. That will work if invoked via cli, entrypoints, or imported as a library.

Comment (alikins) [reply galaxy_importer/main.py]: For example, the logging  state for `python galaxy_importer/main.py  ~/src/automation_hub_collections/netapp.aws-20.2.0.tar.gz` before:

(

```
<--""
   Level WARNING
   |
   o<--"MARKDOWN"
   |   Level NOTSET so inherits level WARNING
   |
   o<--"__main__"
   |   Level INFO
   |   Handler Stream <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
   |     Formatter <__main__.CustomFormatter object at 0x7f65dade84d0>
   |
   o<--[galaxy_importer]
   |   |
   |   o<--[galaxy_importer.ansible_test]
   |   |   |
   |   |   o<--[galaxy_importer.ansible_test.builders]
   |   |   |   |
   |   |   |   o<--"galaxy_importer.ansible_test.builders.local_image_build"
   |   |   |       Level NOTSET so inherits level WARNING
   |   |   |
   |   |   o<--"galaxy_importer.ansible_test.runners"
   |   |       Level NOTSET so inherits level WARNING
   |   |       |
   |   |       o<--"galaxy_importer.ansible_test.runners.base"
   |   |       |   Level NOTSET so inherits level WARNING
   |   |       |
   |   |       o<--"galaxy_importer.ansible_test.runners.openshift_job"
   |   |           Level NOTSET so inherits level WARNING
   |   |
   |   o<--"galaxy_importer.collection"
   |   |   Level NOTSET so inherits level WARNING
   |   |
   |   o<--"galaxy_importer.finder"
   |   |   Level NOTSET so inherits level WARNING
   |   |
   |   o<--"galaxy_importer.loaders"
   |   |   Level NOTSET so inherits level WARNING
   |   |
   |   o<--[galaxy_importer.utils]
   |       |
   |       o<--"galaxy_importer.utils.spdx_licenses"
   |           Level NOTSET so inherits level WARNING
   |
   o<--"requests"
   |   Level NOTSET so inherits level WARNING
   |   Handler <NullHandler (NOTSET)>
   |
   o<--"urllib3"
       Level NOTSET so inherits level WARNING
       Handler <NullHandler (NOTSET)>
       |
       o<--"urllib3.connection"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.connectionpool"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.poolmanager"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.response"
       |   Level NOTSET so inherits level WARNING
       |
       o<--[urllib3.util]
           |
           o<--"urllib3.util.retry"
               Level NOTSET so inherits level WARNING
```

The logging state with pr ('after'):

```
<--""
   Level WARNING
   |
   o<--"MARKDOWN"
   |   Level NOTSET so inherits level WARNING
   |
   o<--"galaxy_importer"
   |   Level INFO
   |   Handler Stream <_io.TextIOWrapper name='<stdout>' mode='w' encoding='UTF-8'>
   |     Formatter <__main__.CustomFormatter object at 0x7f8fee865750>
   |   |
   |   o<--[galaxy_importer.ansible_test]
   |   |   |
   |   |   o<--[galaxy_importer.ansible_test.builders]
   |   |   |   |
   |   |   |   o<--"galaxy_importer.ansible_test.builders.local_image_build"
   |   |   |       Level NOTSET so inherits level INFO
   |   |   |
   |   |   o<--"galaxy_importer.ansible_test.runners"
   |   |       Level NOTSET so inherits level INFO
   |   |       |
   |   |       o<--"galaxy_importer.ansible_test.runners.base"
   |   |       |   Level NOTSET so inherits level INFO
   |   |       |
   |   |       o<--"galaxy_importer.ansible_test.runners.openshift_job"
   |   |           Level NOTSET so inherits level INFO
   |   |
   |   o<--"galaxy_importer.collection"
   |   |   Level NOTSET so inherits level INFO
   |   |
   |   o<--"galaxy_importer.finder"
   |   |   Level NOTSET so inherits level INFO
   |   |
   |   o<--"galaxy_importer.loaders"
   |   |   Level NOTSET so inherits level INFO
   |   |
   |   o<--"galaxy_importer.main"
   |   |   Level NOTSET so inherits level INFO
   |   |
   |   o<--[galaxy_importer.utils]
   |       |
   |       o<--"galaxy_importer.utils.spdx_licenses"
   |           Level NOTSET so inherits level INFO
   |
   o<--"requests"
   |   Level NOTSET so inherits level WARNING
   |   Handler <NullHandler (NOTSET)>
   |
   o<--"urllib3"
       Level NOTSET so inherits level WARNING
       Handler <NullHandler (NOTSET)>
       |
       o<--"urllib3.connection"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.connectionpool"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.poolmanager"
       |   Level NOTSET so inherits level WARNING
       |
       o<--"urllib3.response"
       |   Level NOTSET so inherits level WARNING
       |
       o<--[urllib3.util]
           |
           o<--"urllib3.util.retry"
               Level NOTSET so inherits level WARNING
```

Comment (alikins) [reply galaxy_importer/main.py]: That said, if we ignore the case of importing `galaxy_import.main` from another app, getLogger() is good for an app.

(Original motivation for change was when I needed to be able to import galaxy_import.main from another app, which granted isn't a normal use)

Comment (cutwater) [reply galaxy_importer/main.py]: Why are you calling it as `galaxy_importer/main.py` in the first place?
It should not be used this way and normally python packages should not be used this way.

1. The documentation states it has to be called via `python -m galaxy_importer.main`, which in this case will give you appropriate `__name__` value.
2. This package lacks entry_points:console_scripts section. Normally executable wrappers are generated during package installation.
3. While `python -m galaxy_importer.main` is acceptable, probably it'd be better to refactor it as and create `__main__.py` to allow to be called as `python -m galaxy_importer`.
4. If previous item is done, then changing logger name could make sense, but only if any logic is present in `__main__.py`.

Closing this PR for now.
